plugins/generic.js

```diff
@@ -34,6 +39,7 @@ class Generic {
     this.currentSlide = 1;
     this.isNextSlideDetected = false;
     this.key = this.options.key || exports.options.key.default;
+    this.cssMedia = this.options.cssMedia || exports.options.cssMedia.default;

```

Comment:
I still wonder whether we should define that option globally. We can leave it in the generic plugin and promote it later if needed.